rust/arrow/src/util/integration_util.rs

```diff
@@ -449,12 +449,10 @@ impl ArrowJsonBatch {
                     for i in 0..col.len() {
                         if col.is_null(i) {
                             validity.push(1);
```

Comment:
This is unrelated, but unless I'm reading this incorrectly (likely am), I would have thought that we'd `validity.push(0)` here for null values.